### PR TITLE
updates to aggregated_somatic_mutation node leeuwenhoek

### PR DESCRIPTION
**Pull request by bullenc**
https://gdc-ctds.atlassian.net/browse/DICT-444 + alphabetizing

### CODE REVIEW

**Review by ebusygina — APPROVED** [review]

File: src/gdcdictionary/schemas/aggregated_somatic_mutation.yaml [hunk 1]
```diff
@@ -91,6 +91,10 @@ properties:
       - WGS
       - WXS
 
+  platform:
+    $ref:
+      - "read_group.yaml#/properties/platform"
```
Comment (wteouchicago) [inline src/gdcdictionary/schemas/aggregated_somatic_mutation.yaml:96]: cool! i didn't know we could do that for non-terms things!

**Review by wteouchicago — APPROVED** [review]

**Review by wwysoc2 — APPROVED** [review]
👍

File: src/gdcdictionary/schemas/aggregated_somatic_mutation.yaml (hunk 1 above)
Comment (wwysoc2) [reply src/gdcdictionary/schemas/aggregated_somatic_mutation.yaml:96]: Yeah, I feel like we could do this for experimental strategy, but that's for another release.

**Review by ebusygina — APPROVED** [review]

**Review by stilesj-uchicago — APPROVED** [review]
lgtm

**Pull request merged**